### PR TITLE
Updated druid version to 31.0.0

### PR DESCRIPTION
**Pull request by vivek807**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -63,7 +63,7 @@
     </developers>
 
     <properties>
-        <druid.version>29.0.0</druid.version>
```
Comment (nozjkoitop) [inline pom.xml:66]: I'd keep them separate, in case we'll need to add patches here

**Review by nozjkoitop — CHANGES_REQUESTED** [review]
1 comment left, looks good otherwise

File: pom.xml (hunk 1 above)
Comment (vivek807) [reply pom.xml:66]: Done

**Review by nozjkoitop — APPROVED** [review]
LGTM

**Pull request merged**